FEXCore/Source/Interface/Config/Config.cpp

```diff
@@ -230,27 +230,25 @@ void Load() {
   }
 }
 
-fextl::string ExpandPath(const fextl::string& ContainerPrefix, fextl::string PathName) {
+fextl::string ExpandPath(const fextl::string& ContainerPrefix, const fextl::string& PathName) {
   if (PathName.empty()) {
     return {};
   }
 
-
   // Expand home if it exists
   if (FHU::Filesystem::IsRelative(PathName)) {
     fextl::string Home = getenv("HOME") ?: "";
     // Home expansion only works if it is the first character
     // This matches bash behaviour
     if (PathName.at(0) == '~') {
-      PathName.replace(0, 1, Home);
-      return PathName;
+      return fextl::fmt::format("{}{}", Home, std::string_view(PathName).substr(1));
```

Comment:
Allow reuse of memory allocated by `Home`:
```suggestion
      Home.append(PathName.begin() + 1, PathName.end());
      return Home;
```